gen3-integration-tests/services/fence.py

```diff
@@ -345,3 +352,93 @@ def get_client_id_secret(self, client_name):
         client_info = clients_dict[client_name].split(",")
         client_id, client_secret = client_info[0], client_info[1]
         return client_id, client_secret
+
+    def link_google_account(self, user: str, expires_in: int = None):
+        """Links google account with user account with expiration if provided"""
+        auth = Gen3Auth(refresh_token=pytest.api_keys[user], endpoint=self.BASE_URL)
+        headers = {
+            "Content-Type": "application/json",
+        }
+        if expires_in is not None:
+            url = f"{self.GOOGLE_LINK_REDIRECT}&expires_in={expires_in}"
+        else:
+            url = self.GOOGLE_LINK_REDIRECT
+        logger.info(f"URL: {url}")
```

Comment:
Do we want to print the Google link? Would it be a security concern?

Reply:
No we dont want to print the google link! I will move it to debug in next commit